scalafmt-benchmarks/src/resources/scala-js/GenJSCode.scala

```diff
@@ -1943,7 +1943,7 @@ abstract class GenJSCode extends plugins.PluginComponent
     }
 
     /** Gen jump to a label.
-     *  Most label-applys are caught upstream (while and do..while loops,
+     *  Most label-applies are caught upstream (while and do..while loops,
```

Comment:
It's possible that this should be:
```suggestion
     *  Most `label-apply`s are caught upstream (while and do..while loops,
```